packages/client/src/workqueue/reducer.ts

```diff
@@ -407,16 +400,10 @@ async function getWorkqueueData(
       initialSyncDone: false
     }
   }
-  if (isFieldAgent) {
-    return mergeWorkQueueData(
-      state,
-      ['reviewTab', 'rejectTab'],
-      currentUserData && currentUserData.declarations,
-      workqueue
-    )
-  }
+
   return mergeWorkQueueData(
     state,
+    // @TODO: Test that this works with field agents still
     ['inProgressTab', 'notificationTab', 'reviewTab', 'rejectTab'],
     currentUserData && currentUserData.declarations,
     workqueue
```

Comment:
The fetching of workqueues has been unified to always try and fetch all the records. Test that does this work with field agents. 

**If not:** 
- [ ] design a way to allow all users to fetch the workqueues, but gateway should return empty arrays if the user has no access to the records.

Reply:
It works fine without filtering these, as the backend allows searchEvents to all these tabs. This will be reworked in configurable workqueues, where the workqueues are fetched when opened per-workqueue.